index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Graffiti bootcamp - project 3</title>
+  <!-- dont forget to add a css file and link it here! -->
+  <link rel="stylesheet" href="style.css">
+  <!-- Google fonts-->
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans|Raleway:300,400" rel="stylesheet">
+  <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+</head>
+
+<body>
+  <header>
+    <!-- video or image as a header is cool :) -->
+    <div id=hero>
+      <video autoplay loop muted preload="auto" id="graffitiVideo">
+        <source src="./videos/compressed_graffiti.mp4" type="video/mp4">
+        Your browser doesn't support HTML5 video.
+      </video>
```

Comment:
Consider adding a `<img ` fallback

Sometimes the "Your browser doesn't support HTML5 video." can be replaced by a static image and the user will remain unaware